src/panels/lovelace/common/data.ts

```diff
@@ -38,3 +38,12 @@ export const updateCardConfig = (
     card_config: config,
     format: configFormat,
   });
+
+export const deleteCard = (
```

Comment:
Can we do an inbetween PR to move this file to `src/data/lovelace.ts` (like we have for other components)